apps/www/lib/translations.json

```diff
@@ -1102,15 +1102,23 @@
     },
     {
       "key": "Account/Access/Campaigns/Form/title/0",
-      "value": "Zugriff teilen"
+      "value": "Gäste einladen"
     },
     {
       "key": "Account/Access/Campaigns/Form/title/other",
       "value": "Einen weiteren Zugriff teilen"
     },
```

Comment:
you also want to change this title or eliminate the `t.pluralize` with `campaign.slots.used` in the access campaign form.

Reply:
I don't quite understand this.